seqr/utils/elasticsearch/utils.py

```diff
@@ -29,6 +30,11 @@ def get_es_client(timeout=60, **kwargs):
     }
     if ELASTICSEARCH_CREDENTIALS:
         client_kwargs['http_auth'] = ELASTICSEARCH_CREDENTIALS
+    if ELASTICSEARCH_PROTOCOL:
+        client_kwargs['scheme'] = ELASTICSEARCH_PROTOCOL
+    if ELASTICSEARCH_CA_PATH:
+        ssl_context = create_default_context(cafile=ELASTICSEARCH_CA_PATH)
```

Comment:
how expensive an operation is this? search is already on the slower side, so if this is even a bit slow it might be instead worth running this once in settings.py when the app starts up

Reply:
I didn't notice it taking any perceptible amount of time when I was figuring out how to use it. I don't know that we really have the instrumentation in our monitoring to make a _real_ determination of whether it's adding slowness.

Looking through the docs, the function is basically establishing a group of security settings (i.e. 'how do i verify the server's certificate and what is my minimum tls version?') to apply to a connection. Given that all of our connections serve the same purpose of connecting to elasticsearch, it seems reasonable to re-use that context a little more generally, especially if we're worried about performance of the search itself.

I'll try moving it and see if things still work as expected.